docs/api/rundeck-api.md

```diff
@@ -1069,24 +1069,318 @@ Response:
+  * Accepts: Any `string`
+* `value` Required
+  * The value for the desired config `key`.
+  * Accepts: Any `string`
+* `strata` Optional
+  * Whether the config should apply to the current server (`Server`), or to all servers in the cluster (`default`).
+  * Accepts: `Server`, `default`
+  * Default: `default`
+
+Example JSON
+```json
+[
+  {
+    "key": "myCustomConfig",
+    "value": "newValueForCustomConfig",
+    "strata": "default"
+  },
+  {
+    "key": "myNewCustomConfig",
+    "value": "valueOfNewCustomConfig",
+    "strata": "default"
+  }
+]
+```
+
+**Responses**
+
+    200 OK
+
+All configs were successfully saved or updated. A payload reflecting save or creation status is returned.
+
+Headers  
+`Content-Type: application/json`:
+
+Body
+```json
+{
+  "msg":"Saved",
+  "created": [
+    {
+      "key": "myNewCustomConfig",
+      "value": "valueOfNewCustomConfig",
+      "strata": "default"
+    }
+  ],
+  "updated": [
+    {
+      "key": "myCustomConfig",
+      "value": "newValueForCustomConfig",
+      "strata": "default"
+    }
+  ]
+}
+```
+---
+    400 Bad Request
+
+Some or all configs were formatted in an invalid way and were ignored. Check your request data and try again.
+
+---
+    401 Unathorized
+
+API Key is associated with a user or role with insufficent permissions to perform the action.
+
+Headers  
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "msg":"Unauthorized"
+}
+```
+
+### Delete a Single Config ####
+    DELETE /api/36/config/delete
+
+Delete a single config by key and strata.
+
+**Request:**
+
+Headers
+* `Content-Type` Required
+  * Accepts: `application/json`
+
+Body
+* `key` Required
+  * Represents the config to be deleted.
+  * Accepts: Any `string`
+* `strata` Optional
+  * Whether the config should apply to the current server (`Server`), or to all servers in the cluster (`default`).
+  * Accepts: `Server`, `default`
+  * Default: `default`
+
+Example JSON
+```json
+  {
+    "key": "myCustomConfig",
+    "strata": "default"
+  }
+```
+
+**Response**
+
+    204 No Content
+
+The config was deleted successfully.
+
+---
+    401 Unathorized
+
+API Key is associated with a user or role with insufficent permissions to perform the action.
+
+Headers  
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "msg":"Unauthorized"
+}
+```
+
+---
+    404 Not Found
+
+A config with the input config key was not found.
+
+Headers  
+`Content-Type: application/json`
+
+Body
+```json
+{
+  "error":true,
+  "apiversion":40,
+  "errorCode":"rundeckpro.config.api.error.not.found",
+  "message":"System Configuration Not Found"
+}
+```
+
+### List All Current Configurations ####
+    GET api/36/config/list
+List all existing configs and their properties.
 
 **Request:**
 
+Headers
+* `Content-Type` Optional
+  * Accepts: `application/json`
+
+**Responses:**
+
+    200 OK
+
+Call was successful. A list of all configs is returned.
```

Comment:
I think we need to clarify that this is all configs stored using Configuration Management.  For a list of "All Configs" reference System Report.

Reply:
Added as a note